### ISSUE TITLE
Audio applet menu default size too small

### ISSUE THREAD

**Issue by singerintherain** [EliverLara/Nordic#284]
Hello,

This is probably not an issue per se, but I didn't find an answer in mate/panel manuals. Is there a way to resize this menu so that it doesn't have to scroll? After scrolling down, when I try to click audio settings, it scrolls up again, so I can't reach audio setting from this menu.

The menu is full size in another themes.

![Zrzut ekranu z 2023-11-16 13-56-50](https://github.com/EliverLara/Nordic/assets/151039197/c2e5ad20-42f6-4db9-a650-3faa1b34b481)

**Comment by EliverLara** [comment]
Hi, Are you using the latest version of the theme? since it seems to be reported and fixed some time ago in #214

**Comment by singerintherain** [comment]
Yes, I'm using the latest version. Please tell me if you need any more details (and how to find them).

**Comment by EliverLara** [comment]
I've tested the theme and the issue is not present at least on my side:

![Captura de pantalla de 2024-02-12 17-04-10](https://github.com/EliverLara/Nordic/assets/9838260/8356e67d-1ce8-4a0a-9c89-8b54c03cc74c)


Could you please try using the latest version published on [gnome-look](https://www.pling.com/p/1267246/)

**Comment by singerintherain** [comment]
Hello, I installed the current version and it doesn't have the issue. Thanks!

**Comment by EliverLara** [comment]
@singerintherain Nice to know that!
